### DISCUSSION TITLE
Enhancement: Disable selected preset (use bare model after preset has been clicked on)

### DISCUSSION THREAD

**Discussion by erickrf** [danny-avila/LibreChat#4273] [Feature Requests & Suggestions]
### What features would you like to see added?

If I select a preset for a new chat, I don't see a way to unselect it. I may select other presets instead, but I don't see how to talk to the bare model.

### More details

In the preset selector, there should be an "unselect" or "bare model" or something like this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
a preset cannot be "unselected" because it only applies specific settings to the current settings. this can be cleared by hitting "new chat."

I do agree that there should be a "reset" settings in parameters, this would be more aligned with the current design.

**Reply by erickrf** [reply]
True, I hadn't realized that. 

Also, after choosing a preset and clicking again on the preset selector, it still shows "No default preset active." This is a bit confusing.
